apscheduler/triggers/interval.py

```diff
@@ -20,12 +20,14 @@ class IntervalTrigger(BaseTrigger):
     :param datetime|str start_date: starting point for the interval calculation
     :param datetime|str end_date: latest possible date/time to trigger on
     :param datetime.tzinfo|str timezone: time zone to use for the date/time calculations
+    :param int|None jitter: randomize the job execution time. Job will be advanced or delayed by
```

Comment:
Should probably be rephrased as "advance or delay the job execution by `jitter` seconds"